src/pages/Menu.js

```diff
@@ -1,7 +1,20 @@
 import MenuItem from "../components/MenuItem";
 
-export default function Menu({ menuItems, onUpdateQuantity: updateQuantity }) {
-  return menuItems.map((item) => {
+export default function Menu({
+  menu,
+  order,
+  onUpdateQuantity: updateQuantity,
```

Comment:
I think you receive a prop only in form of a key and not a value as well, right?